staging/src/kubevirt.io/client-go/kubevirt/typed/core/v1/fake/fake_virtualmachine_expansion.go

```diff
@@ -107,7 +107,7 @@ func (c *FakeVirtualMachines) PortForward(name string, port int, protocol string
 
 func (c *FakeVirtualMachines) ObjectGraph(ctx context.Context, name string, objectGraphOptions *v1.ObjectGraphOptions) (v1.ObjectGraphNode, error) {
 	obj, err := c.Fake.
-		Invokes(fake2.NewPutSubresourceAction(virtualmachinesResource, c.ns, "objectgraph", name, objectGraphOptions), nil)
+		Invokes(fake2.NewGetSubresourceAction(virtualmachinesResource, c.ns, "objectgraph", name, objectGraphOptions), nil)
```

Comment:
Is that a generated change?

Reply:
I was wondering why it changes from Put to Get

Reply:
The original PUT was wrong. ObjectGraph is a GET request